code/find/lib/main.js

```diff
@@ -1,12 +1,6 @@
 import data from "./spaceResorts.js";
-import sorts from "./sorts.js";
 
-export default {
-  findSpaceResorts: findSpaceResorts,
-  selectSpaceResorts: selectSpaceResorts,
-};
-
-function findSpaceResorts(name, planet, searchCriteria, $vivContext) {
+export default function findResorts(name, planet, searchCriteria, $vivContext) {
```

Comment:
why rename? can we keep the original name?

Reply:
@kangawoo44 the problem was that the function was being used in another file, and it had the same name. I can rename the function that is calling "findSpaceResorts" instead, like in the image:


![image](https://user-images.githubusercontent.com/102233419/232882765-bd19ccec-861f-4674-9310-fd4c93348914.png)

Reply:
Sure that seems fine. Another thing we could do is we don't declare a default export for main.js and import all as `main` and keep the same syntax as before

Reply:
@kangawoo44 Seems like we are not allowed to leave the file without a default export:
![image](https://user-images.githubusercontent.com/102233419/232899548-46d15beb-b474-4fea-874d-9d7db078c18e.png)

Reply:
oh I see. we can still import all as main.

Reply:
Yes, importing everything as main now

Reply:
I liked your idea too btw 😄